src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -484,4 +484,26 @@ class JavaMigrations {
     ).validate().insert()
     Seq(Linux32, Linux64, MacOSX, Windows).foreach(removeVersion("java", "12.0.1-librca", _))
   }
+  
+  @ChangeSet(order = "172", id = "172-add_adoptopenjdk-hs_8_0_222", author = "jschalanda")
+  def migrate172(implicit db: MongoDatabase) = {
+    List(
+      Version("java", "8.0.222.hs-adpt", "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u222-b10/OpenJDK8U-jdk_x64_linux_hotspot_8u222b10.tar.gz", Linux64, Some(AdoptOpenJDK)),
+      Version("java", "8.0.222.hs-adpt", "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u222-b10/OpenJDK8U-jdk_x64_mac_hotspot_8u222b10.tar.gz", MacOSX, Some(AdoptOpenJDK)),
+      Version("java", "8.0.222.hs-adpt", "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u222-b10/OpenJDK8U-jdk_x64_windows_hotspot_8u222b10.zip", Windows, Some(AdoptOpenJDK)))
+      .validate()
+      .insert()
+    Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "8.0.212.hs-adpt", _))
+  }
+
+  @ChangeSet(order = "173", id = "173-add_adoptopenjdk-hs_8_0_222", author = "jschalanda")
```

Comment:
Possible copy/paste error in `id` of ChangeSet... should be 173-add_adoptopenjdk-**j9**_8_0_222

Reply:
Good catch, thanks!